apps/prairielearn/src/pages/instructorAssessmentInstance/instructorAssessmentInstance.js

```diff
@@ -47,9 +47,24 @@ const DateDurationResultSchema = z.object({
 });
 
 const InstanceQuestionRowSchema = InstanceQuestionSchema.extend({
+  instructor_question_number: z.string().nullable(),
+  manual_rubric_id: IdSchema.nullable(),
+  max_auto_points: z.number().nullable(),
+  max_manual_points: z.number().nullable(),
+  max_points: z.number().nullable(),
   modified_at: z.string(),
   qid: z.string().nullable(),
+  question_id: IdSchema,
   question_number: z.string().nullable(),
```

Comment:
Is this actually nullable?

Reply:
No, this should never be null. Removed `.nullable()` in commit [7a88b09](https://github.com/PrairieLearn/PrairieLearn/pull/9250/commits/7a88b098dd62ef42b7196aa4cdafa015e6675a30).